styleguide/css/main.scss

```diff
@@ -47,9 +43,50 @@ $medium-gray: #666;
 }
 
 .openbtn {
-  background-color: $dark-gray;
+  background-color: $gray-dark;
   left: 0;
   padding: 10px;
   position: absolute;
   top: 0;
 }
+
+.anchored-text {
```

Comment:
Nesting panels created an issue where the help text for a pattern was actually getting in the way and creating a misrepresentation of what the mini panel would look like. As a result I opted to just create little icons that people can click on to get more details about the pattern, and that gets placed inside of `anchored-text`.  Since this is zoolander-only behavior, I placed it in main.scss to avoid adding unnecessary code to the codebase.